parachains/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Get Started
+description: Practical examples and tutorials for building and deploying parachains on Polkadot, from launching to customization and cross-chain messaging.
```

Comment:
```suggestion
description: Practical examples and tutorials for building and deploying Polkadot parachains, covering everything from launch to customization and cross-chain messaging.
```

I think "launching to customization" sounded a little off. I don't know anyways, my AI friend generated something similar 🤷‍♀️